sibyl/resources/computing.py

```diff
@@ -98,12 +101,22 @@ def get_entity_table(eid, row_id):
     return pd.DataFrame(get_features_for_row(entity.features, row_id), index=[0])
 
 
-def get_entities_table(eids, row_id, all_rows=False):
+def get_entities_table(eids, row_ids, all_rows=False):
     if not all_rows:
-        entities = [
-            dict(get_features_for_row(entity.features, row_id), **{"eid": entity.eid})
-            for entity in schema.Entity.objects(eid__in=eids)
-        ]
+        if row_ids is None:
+            entities = [
+                dict(get_features_for_row(entity.features, row_ids), **{"eid": entity.eid})
```

Comment:
I may be missing something here, but its not clear to me what the functionality of get_features_for_row would be if row_ids is None

Reply:
I also think this logic tree would be a bit more readable if you switched the order of the last two conditions. That way the order goes [one row unspecified, one row, multiple rows], rather than switching between one and multiple rows

Reply:
when row_ids is None (the case for all old functionality). `entities` would contain the first row of each entity.

Reply:
Oh I see, I missed the difference between the two functions. Can we switch `get_features_for_rows` to `get_features_for_multiple_rows` or something, to make it more immediately different from `get_feature_for_row`?